### PR TITLE
Adding script to generate element blocks for 1 of 1 regtest

### PR DESCRIPTION
**Pull request by Christewart**
Adds a simple python script to generate blocks for elements on regtest

### CODE REVIEW

**Comment by cdecker** [comment]
You might want to wrap the function call into 

```
if __name__ == "__main__":
    generate_n_blocks(100)
```
to make the module importable :smiley:

**Comment by Christewart** [comment]
I'll add this later tonight, I also justt placed the script in the top level directory, should it be moved somewhere else?

**Comment by Christewart** [comment]
Added changes suggested by @cdecker , also adding command line arguments so we can now use it like so: 
```
python generate_element_blocks.py --num=100 --datadir=$BETADATADIR
```
where `num` is the number of blocks to generate

File: generate_element_blocks.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+def generate_n_blocks(n,datadir): 
+    for i in range(0,n): 
+        sign_block(datadir)
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--num')
+    parser.add_argument("--datadir")
+    datadir="~/.bitcoin/elements.conf"
```
Comment (jtimon) [inline generate_element_blocks.py]: Not sure if you want to do
datadir="~/.bitcoin/elements/"
conf="~/.bitcoin/elements/elements.conf"

**Comment by jtimon** [comment]
Needs rebase (just remove all the commits except yours and I think it should rebase without conflicts).

**Comment by jtimon** [comment]
Actually, no, it needs to be reopened as a PR to master instead of refresh, sorry for the oversight.

**Pull request closed without merging**